src/Proto.Remote.GrpcCore/Remote.cs

```diff
@@ -0,0 +1,113 @@
+﻿// -----------------------------------------------------------------------
+//   <copyright file="Remote.cs" company="Asynkron AB">
+//       Copyright (C) 2015-2020 Asynkron AB All rights reserved
+//   </copyright>
+// -----------------------------------------------------------------------
+
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using Grpc.Core;
+using Grpc.Health.V1;
+using Grpc.HealthCheck;
+using JetBrains.Annotations;
+using Microsoft.Extensions.Logging;
+
+namespace Proto.Remote
+{
+    [PublicAPI]
+    public class Remote : IRemote

```

Comment:
Namespace / class cannot clash, this will collide with the GrpcNet implementation..

Reply:
The idea was to be backward compatible. The client project will only have to reference one Implementation.
But I can change that to be SelfHostedGrpcCoreRemote and SelfHostedGrpcNetRemote or change the namespaces to be Proto.Remote.GrpcNet and Proto.Remote.GrpcCore.

Reply:
Doing it with the namespace could be enough I think. Perhaps GrpcNetRemote / GrpcCoreRemote for the classes?